lang/cs/Org.Apache.REEF.IMRU/OnREEF/Driver/IMRUDriver.cs

```diff
@@ -641,6 +653,10 @@ public void OnNext(IFailedTask failedTask)
                             TryRecovery();
                             break;
 
+                        case SystemState.Fail:
```

Comment:
This is not expected case. Because when we set system state to Fail, we throw exception.